packages/sku/entry/csp.js

```diff
@@ -53,18 +71,18 @@ export default function createCSPHandler({
       );
     }
 
-    if (
-      process.env.NODE_ENV !== 'production' &&
-      !valid(script, { script: true })
-    ) {
+    if (process.env.NODE_ENV !== 'production' && !valid(script)) {
       console.error(`Invalid script passed to 'registerScript'\n${script}`);
     }
 
-    parse(script, { script: true })
+    parse(script, { blockTextElements: { script: true } })
       .querySelectorAll('script')
       .forEach(processScriptNode);
   };
```

Comment:
The `script` property is now under `blockTextElements`. It actually defaults to `true`, so we don't _really_ need it, but probably best to be explicit. Unit tests were added to ensure this change didn't break anything.

Reply:
Should we be consistent and add it to the `valid` call [on line 74]?

[on line 74]: https://github.com/seek-oss/sku/pull/981/files#diff-890cc153983cf273d29b3e92fe5218d82a8661da2cbde0582d32b4f8a445ce99R74

Reply:
Have removed `script` as it's the `true` by default. https://github.com/seek-oss/sku/pull/981/commits/ea6525c34ece493695aa167ccc810c60cdf75d97